web/vital_records/templates/vital_records/request/confirm.html

```diff
@@ -58,6 +58,12 @@ <h3 class="md">Order information</h3>
     <form method="post" class="m-b-40 p-b-lg">
         {% csrf_token %}
         {{ form.as_p }}
-        <button type="submit" class="btn btn-primary m-b-sm">Submit your order</button>
+        {% if not already_submitted %}
+            <button type="submit" class="btn btn-primary m-b-sm">Submit your order</button>
+        {% else %}
+            <p>
+                <span class="ca-gov-icon-check-fill pe-2" aria-hidden="true"></span>You’ve already submitted this order. To start a new order <a href="{% url "vital_records:request_start" %}">request another record</a>.
```

Comment:
It makes the most sense for the link to go to `request_start` because that's the view where a new request is created.

It would get complicated if we wanted it to link to somewhere else. For example if it were to link to `request_type`, we would have to somehow create a new request and fill out the `fire` field using the old request's `fire`? 

I strongly advocate for it linking to `request_statement`.

Reply:
> I strongly advocate for it linking to `request_statement`.

Did you mean `request_start`, as stated in the previous paragraph and what the code has currently? Because I agree, `request_start` makes the most sense here.

Reply:
Oops, yes! Thanks